controllers/configurationpolicy_controller.go

```diff
@@ -1538,8 +1538,19 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(
 			// it is a must have and it does exist, so it is compliant
 			compliant = true
 			if strings.EqualFold(string(remediation), string(policyv1.Enforce)) {
-				statusUpdateNeeded = createStatus("", obj.gvr.Resource, compliantObject, obj.namespaced, obj.policy,
-					obj.index, compliant, true)
+				if updatedObj {
+					// object updated in checkAndUpdateResource, send event
+
+					log.Info("Updated must have object", "resource", obj.gvr.Resource, "name", obj.name)
+					reason := "K8s update success"
```

Comment:
How about this?
```suggestion
					reason := "Object was updated"
```

Reply:
@mprahl we have "K8s creation success" and "K8s deletion success" for the creation and deletion conditions- I'd prefer to keep it this way to keep it consistent if that's ok with you

Reply:
That's a good point. Let's keep it the way you have it in the PR.